packages/core/src/__tests__/ceramic-feed.test.ts

```diff
@@ -18,8 +19,18 @@ describe('Ceramic feed', () => {
   let ceramic1: Ceramic
   let ceramic2: Ceramic
   beforeAll(async () => {
-    ipfs1 = await createIPFS()
-    ipfs2 = await createIPFS()
+    ipfs1 = await createIPFS({
+      rust: {
```

Comment:
should we just make this the default behavior of the `createIPFS` helper function so we don't need to pass these values in manually all over the place?  I'm pretty sure `createIPFS` is testing-only code already anyway